contracts/token/SRC20.sol

```diff
@@ -441,6 +336,32 @@ contract SRC20 is ISRC20, ISRC20Owned, ISRC20Managed, SRC20Detailed, Featured,
         return true;
     }
 
+    function transfer(address to, uint256 value) external returns (bool) {
+        if (_restrictions != ITransferRules(0)) {
+            require(_restrictions.authorize(msg.sender, to, value), "Transfer not authorized");
+
+            _restrictions.doTransfer(msg.sender, to, value);
+        } else {
+            _transfer(msg.sender, to, value);
+        }
+
+        return true;
+    }
+
+    function transferFrom(address from, address to, uint256 value) public returns (bool) {
+        if (_restrictions != ITransferRules(0)) {
```

Comment:
Again, **Notice**: If SRC20 does not have on-chain rules (`_restrictions == 0`), someone can call this transferFrom call and avoid off-chain rules.

Reply:
Same as above, nothing to add more.